tests/php/bootstrap.php

```diff
@@ -56,7 +56,7 @@
 
 if ( false === function_exists( 'wp_cache_is_enabled' ) ) {
 	/**
-	 * "Mocking" function so that it exists and Jetpack_Sync_Actions will load Jetpack_Sync_Module_WP_Super_Cache
+	 * "Mocking" function so that it exists and Jetpack_Sync_Actions will load WP_Super_Cache
```

Comment:
Feels like this should have the fully qualified name, otherwise it's meaningless (because we don't really have a `use` for it here)?
```suggestion
	 * "Mocking" function so that it exists and Jetpack_Sync_Actions will load Automattic\Jetpack\Sync\Modules\WP_Super_Cache
```

Reply:
use fully qualified name